spec/models/account_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe Account, :type => :model do
+  it "is valid with valid attributes" do
+    account = Account.new(name: Faker::Company.name)
```

Comment:
It is better to declare a `let` variable above and use it here
`let(:name) { Faker::Company.name }`

Reply:
(This should be outside the `it` example)

Reply:
http://www.betterspecs.org/#let